assignments/arrays.js

```diff
@@ -61,36 +66,219 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 //     arr[i]; // 1,2,3,4
 // }
 
+// let inventory = [
+//     {"id":1,
+//     "car_make":"Lincoln",
+//     "car_model":"Navigator",
+//     "car_year":2009},
+
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
 
+console.log("==== Challenge 1 ==== \n");
+for(let i = 0; i < inventory.length; i++) {
+    if(inventory[i].id === 33) {
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}\n` );
+    }
+}
```

Comment:
I like your if statement here! But remember the array index starts at `0` so you'd actually be looking for index `32`.

Reply:
I know, but if this array comes from a database or the array is in a different order or you delete one car before, then not always is going to be in the position #32.

Reply:
In this problem, you're given an array and asked for the 33rd car. The 33rd car, no matter how you mess with the array, will be at index 32.

If it were drawn from a database, you're absolutely right. We typically try to assign ids that are independent of the position in the database to avoid exactly this confusion. (for example, car id #3a could be moved around to different spots on the list)

EXCELLENT thinking on this, Vlad!

Reply:
And on re-reading the problem, it looks like there was some ambiguity in the instructions.  So good job! Do you see how the use cases differ?